src/ucan.ts

```diff
@@ -11,53 +11,103 @@ export * from "./crypto.js"
 
 export type { MultihashDigest, MultibaseEncoder, MultihashHasher }
 
+/** Arbitrary (service / capability specific) information included in a UCAN {@link Body} */
```

Comment:
What do you think about this instead ?

```suggestion
/**
 * Verifiable facts and proofs of knowledge included in a UCAN {@link Body} in order to
 * support claimed capabilities.
 * @see https://github.com/ucan-wg/spec/#324-facts
 */
```